src/test/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlDialectTest.java

```diff
@@ -0,0 +1,101 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.*;
+import static com.exasol.adapter.capabilities.LiteralCapability.*;
+import static com.exasol.adapter.capabilities.MainCapability.*;
+import static com.exasol.adapter.capabilities.PredicateCapability.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.when;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.capabilities.Capabilities;
+import com.exasol.adapter.dialects.BaseQueryRewriter;
+import com.exasol.adapter.dialects.SqlDialect.NullSorting;
+import com.exasol.adapter.dialects.SqlDialect.StructureElementSupport;
+import com.exasol.adapter.jdbc.ConnectionFactory;
+
+@ExtendWith(MockitoExtension.class)
+public class ElasticSearchSqlDialectTest {
```

Comment:
Don't forget to write integration tests later

Reply:
yeah, this is gonna be in a following PR